### PR TITLE
Optimise s3cmd puppet module

### PR DESCRIPTION
**Pull request by nikkonix**

### CODE REVIEW

File: manifests/commands/get.pp [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-     creates => $name,
-     require => Class['s3cmd::config','s3cmd::install'],
-  }
-}
\ No newline at end of file
+    $homedir = $owner ? {
+        root    => '/root',
+        default => "/home/${owner}",
+    }
+    $testvar = $name
```
Comment (Vincent--) [inline manifests/commands/get.pp]: Do we need this testvar variable?

File: manifests/config.pp [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+    $gpg_passphrase,
+    $owner,
 ) {
-  $homedir = $owner ? {
-    root => '/root',
-    default => "/home/$owner",
-  }
+    $homedir = $owner ? {
+        root    => '/root',
+        default => '/home/$owner',
```
Comment (Vincent--) [inline manifests/config.pp]: Shouldn't that use double quotes?

File: manifests/install.pp [hunk 3]
```diff
@@ -1,17 +1,10 @@
-class s3cmd::install (){
-
-  if $osfamily == 'redhat' {
```
Comment (Vincent--) [inline manifests/install.pp:3]: Why did you remove the part of the code related to the osfamily?

**Review by Vincent-- — CHANGES_REQUESTED** [review]

File: manifests/commands/get.pp (hunk 1 above)
Comment (nikkonix) [reply manifests/commands/get.pp]: I tested and forgot this variable. Fixed.

File: manifests/config.pp (hunk 2 above)
Comment (nikkonix) [reply manifests/config.pp]: Fixed.

File: manifests/install.pp (hunk 3 above)
Comment (nikkonix) [reply manifests/install.pp:3]: I removed in the beginning, before this conversation in which we discuss that this module wont service only to us and one day we can request PR to the origin. Restored.

**Review by Vincent-- — APPROVED** [review]

**Pull request merged**